deploy/twingate-operator/values.schema.json

```diff
@@ -47,9 +47,43 @@
             "type": "object",
             "default": {},
             "title": "The twingateOperator Schema",
-            "required": ["apiKey", "network", "remoteNetworkId"],
+            "oneOf": [
+                {
+                    "required": ["apiKey", "network", "remoteNetworkId"]
+                },
+                {
+                    "required": ["existingAPIKeySecret", "network", "remoteNetworkId"]
+                }
+            ],
             "properties": {
                 "apiKey": { "type": "string" },
+                "existingAPIKeySecret": {
+                    "type": "object",
+                    "title": "An existing secret with the API Key to be used by the operator. If this is specified, the apiKey field will be ignored.",
```

Comment:
I don't think this is correct. When I pass in both `twingateOperator.existingAPIKeySecret.name` and `twingateOperator.apiKey` the apiKey is not ignored. It throws an error

```
helm template --debug -f values.yaml --set twingateOperator.existingAPIKeySecret.name=mysecret --set twingateOperator.existingAPIKeySecret.key=mykey --set twingateOperator.apiKey=test --set twingateOperator.network=test --set twingateOperator.remoteNetworkId=test .

Error: values don't meet the specifications of the schema(s) in the following chart(s):
twingate-operator:
- twingateOperator: Must validate one and only one schema (oneOf)
```

Reply:
You're right. I learned how to use `oneOf` :)
will update